front/assets/js/git_integration/components/card.tsx

```diff
@@ -22,7 +22,7 @@ interface CardProps {
 
 export const Card = (props: CardProps) => {
   return (
-    <div className={`ph3 pv2 mv2 ${props.lastItem ? `` : `bb b--black-075`}`}>
+    <div data-testid="integration-card" className={`ph3 pv2 mv2 ${props.lastItem ? `` : `bb b--black-075`}`}>
```

Comment:
I don't like the name of the `data-testid` attribute. WDYT about changing it to something like `data-element-id`? 🤔

Reply:
the `data-testid` seems to be the [standard practice](https://www.educative.io/answers/what-is-the-data-testid-attribute-in-testing) for assuring stable DOM selectors in tests.